### PR TITLE
[angular-xmcloud] Introduce angular SXA Image and Banner components

### PR DESCRIPTION
**Pull request by yavorsk**
- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)
- [x] Changelog updated
- [ ] Upgrade guide entry added

## Description / Motivation

Introduce SXA Image and Banner components

## Testing Details

- [ ] Unit Test Added
- [ ] Manual Test/Other (Please elaborate)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/image/image.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<ng-container [ngTemplateOutlet]="variant"></ng-container>
+
+<ng-template #default>
+  <div class="component image {{ styles }}" [attr.id]="id" *ngIf="rendering.fields; else empty">
```
Comment (sc-addypathania) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/image/image.component.html:4]: we can use the `ngClass` directive to improve readability.
Similarly wherever classes are defined.

**Review by sc-addypathania — COMMENTED** [review]
see one comment. Otherwise looks good.

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/image/image.component.html (hunk 1 above)
Comment (yavorsk) [reply packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/image/image.component.html:4]: yea, makes sense, I will update

**Review by sc-addypathania — APPROVED** [review]
👍🏼

**Pull request merged**